docs/01_introduction/README.md

```diff
@@ -238,7 +239,7 @@ The EasyBuild **framework** leverages **easyblocks** to automatically build and
 (scientific) software, potentially including additional **extensions**, using a particular compiler **toolchain**,
 as specified in **easyconfig files**.
 
-EasyBuild ensures that the specified **dependencies** are installed,
+EasyBuild ensures that the specified **dependencies** are met by installing them as required,
```

Comment:
Installing deps is only done when `--robot` is used, by default EasyBuild will only make sure that the dependencies that are required for a particular installation are there before trying to install it.

Reply:
Doh. Of course. That gets clumsy. "EasyBuild ensures that the specified **dependencies** are met upon a successful install,"?

Reply:
Well, it ensures that deps are there before starting the installation. I'll try to reword it.